### ISSUE TITLE
[20220320] Weekly AI ArXiv 만담 - 44회차

### ISSUE THREAD

**Issue by jungwoo-ha** [jungwoo-ha/WeeklyArxivTalk#44]
- News
  - Interspeech 2022 deadline: 3월 21일 / 28일 - 모두들 파이팅입니다.
  - [AI미래포럼 1주년 웨비나: 차기정부 AI 국가전략: 30여개 질문 + 약간의 논쟁](https://www.youtube.com/watch?v=0cKpdZv_2F4)
  - [New GPT-3 Capabilities: Edit & Insert](https://openai.com/blog/gpt-3-edit-insert/)
  - [Stanford AI Index Report 2022](https://hai.stanford.edu/research/ai-index-2022): 다음 주 요거 특집으로 준비해보겠습니다. 
- ArXiv
  - [HyperMixer: An MLP-based Green AI Alternative to Transformers](https://arxiv.org/abs/2203.03691)
    - Transformer 의 O(L^2) 문제 해결을 위해 MLP-Mixer 의 token mixing
    - MLP weight를 generation 하는 Hypernetwork 구조 (이거 잘 안되는데...)
    - 그래서 variable length input에 대한 처리 가능
    - 다양한 NLU task에서 정확도 유지하면서 계산량 이점을 취함 (Transformer 보다 낫고 MLP-Mixer나 gMLP 보다 나은듯)
![image](https://user-images.githubusercontent.com/11782739/159120396-99c7c03f-b696-481d-8881-e3d3b4409d94.png)
![image](https://user-images.githubusercontent.com/11782739/159120464-82fc977d-10b0-438e-8d96-5333caaf4bb3.png)
![image](https://user-images.githubusercontent.com/11782739/159120453-3f44becc-0e00-4a83-9ccd-e9f7ff3f01d3.png)
 
  - [cosFormer: Rethinking Softmax in Attention](https://arxiv.org/abs/2202.08791)
    - SenseTime 에서 나온 Performer 류의 Kernel trick을 이용한 O(N) softmax approx. Transformer (ICLR 2022)
    - non-negative kernel (이라고 해도 $PI$ 대신 ReLU 쓴다는), cosine-based reweighting trick
    - NLU task와 Long-range arena에서 상당히 매력적인 성능을 보여줌. 
    - 참고로 전 수식 많은 논문이 싫어요...
    - https://github.com/OpenNLPLab/cosFormer
![image](https://user-images.githubusercontent.com/11782739/159123824-bc2fe2f6-8d1d-4baf-84f3-1257702dbb22.png)
![image](https://user-images.githubusercontent.com/11782739/159123838-70653046-2989-496a-96ac-2c49cc0200f7.png)
![image](https://user-images.githubusercontent.com/11782739/159123849-e6845d60-3092-4e76-a41d-ab9bc7f28736.png)
   
  - 기타 흥미있는 연구
    - [Scaling Up Your Kernels to 31x31: Revisiting Large Kernel Design in CNNs](https://arxiv.org/abs/2203.06717)
    - [OCR 패키지 from OpenMMLab](https://github.com/open-mmlab/mmocr)
      - https://openmmlab.com/ 
    - [SUPERB-SG: Enhanced Speech processing Universal PERformance Benchmark for Semantic and Generative Capabilities](https://arxiv.org/abs/2203.06849)
      - https://github.com/s3prl/s3prl 
    - [Bamboo: Building Mega-Scale Vision Dataset Continually with Human-Machine Synergy](https://arxiv.org/abs/2203.07845)
      - https://opengvlab.shlab.org.cn/bamboo/home

**Comment by ghlee0304** [comment]
- Arxiv (Audio and Speech Processing)
    - [U-Singer: Multi-Singer Singing Voice Synthesizer that Controls Emotional Intensity](https://arxiv.org/abs/2203.00931) 
        - Arxiv / 한동대 / SVS, Emotion 
    - [SpeechFormer: A Hierarchical Efficient Framework Incorporating the Characteristics of Speech](https://arxiv.org/pdf/2203.03812.pdf)
        - arxiv, INTERSPEECH2022 (submitted) / South China University of Technology, UBTECH Robotics, iFLYTEK Research
        -

**Comment by hollobit** [comment]
### Deep Neural Nets: 33 years ago and 33 years from now
- https://karpathy.github.io/2022/03/14/lecun1989/?fbclid=IwAR0BxDHCUXNfEWidZQw7l-FDOu7quPNdXBx2e10VwXU0VxA_N895EEqBlzM
>33년 전(1989년) 딥러닝 기술과 현재, 그리고 앞으로 33년 뒤(2055년)에 대한 전망 내용
>33년 전(1989년)과 오늘날의 비교 
>  * 거시적으로는 33년 동안 크게 달라진 것이 없다. MLP와 BP 알고리즘의 확장 모델 
>  * 데이터 세트와 모델 파라미터 크기에서는 큰 차이 
>  * 33년전 웍스테이션에서 3일 걸렸던 모델이, 현재는 일반 노트북에서 90초만에 학습 

>33년 후(2055년)에 대한 전망 
>  * 2055 신경망은 더 큰 것을 제외하고 기본적으로 거시적 수준에서 2022 신경망과 동일
>  * 현재의 데이터 세트와 모델 파라미터 크기와 비교해서는 10,000,000배 정도 커질 것으로 전망 
>  * 현재의 최신 GPT-3 모델 정도도, 2055년 노트북에서는 1분 이내 학습시킬 수 있음

### Multimodal Neural Script Knowledge through Vision and Language and Sound (CVPR 2022)
- Page: https://t.co/WkZ5LP3mMs
- Paper: https://t.co/BImN4I84CD
- Demo: https://t.co/8Au1NfMWyV
- Github : https://github.com/rowanz/merlot_reserve/
>- NeurIPS 2021d에서 발표했던 MERLOT:Multimodal Neural Script Knowledge Models의 후속 연구 -https://rowanzellers.com/merlot/
>- MERLOT Reserve는 비디오에 대한 질문에 답하기 위한 새로운 모델
>- 비디오와 질문이 주어지면 MERLOT Reserve는 객관식 옵션 목록에서 정답을 안정적으로 선택할 수 있음
>-MERLOT Reserve는 비디오의 시각적 프레임, 비디오의 자막 및 비디오의 오디오를 공동으로 추론하여 이 예측을 수행
>-사람이 라벨을 붙인 데이터 없이도 많은 간단한 비디오 이해 작업을 성공적으로 수행하는 데 사용할 수 있음
>- 2천만개 youtube 비디오, 10억 프레임의 데이터셋 활용  (2021년 MERLOT 모델은 6백만개 비디오 데이터)
<img src="https://rowanzellers.com/merlotreserve/img/reservefig-modelonly-v2.png">

**Comment by kimyoungdo0122** [comment]
- News
    - ["부총리 참여 'AI 특위' 만들어 생태계 키워야"](https://www.hankyung.com/it/article/2022031728861)
        - AI 미래포럼 뉴스 요약
    - [AI and Human Enhancement: Americans’ Openness Is Tempered by a Range of Concerns](https://www.pewresearch.org/internet/2022/03/17/ai-and-human-enhancement-americans-openness-is-tempered-by-a-range-of-concerns/)
        - Pew Research Center라는 곳에서 실시한 미국인들의 AI에 대한 인식 조사
        - AI application 중 미국 경찰의 안면 인식, 잘못된 정보를 걸러내려는 소셜 미디어 회사들의 알고리즘, 무인자동차, 인류의 육체 진화(?), 의료 및 생명공학에서의 AI 활용, 신체에 장착하는 로봇 골격(?) 등에 대해 조사
        - 성별, 연령, 학력 등 인구통계학적인 차이가 꽤 있어서 흥미로웠습니다.
        - Summary
        - ai의 긍정적인 효과를 기대하거나, 아니면 부정적인 결과를 우려하거나, 불확실성 자체를 의심하거나 등
        - 그럼에도 불구하고 ai가 현재 사회를 보다 더 발전시킬 것이라고 생각
        - 그래서 사람이 하는 것보다 더 높고 까다로운 수준의 기술 안정성 평가가 필요하다 생각
        - 자율 주행 차량에도 운전자가 있어야 한다는 의견이 더 많고, 아예 사람도 없는 무인 차량은 표시해야하는 등 사회에 녹아들기 위해 조치를 촉구하기도 함
![image](https://user-images.githubusercontent.com/42539153/159163685-a0899494-8bb4-48d4-bbfd-2c650e3c951a.png)

        -

**Comment by nick-jhlee** [comment]
- Can Neural Nets Learn the Same Model Twice? Investigating Reproducibility and Double Descent from the Decision Boundary Perspective (CVPR 2022)
 - Univ of Maryland (Prof. Tom Goldstein), Rice University, NYU
 - We see empirically that the decision boundaries of a network have strong similarities across runs, (e.g. random init)
 - Decision boundary를 이용한 여러 deep learning에 대한 실험들!
    - double descent
    - label noise
 
![image](https://user-images.githubusercontent.com/38721329/159164955-2b44bcae-b99b-4553-8c61-f4c4452db974.png)

![image](https://user-images.githubusercontent.com/38721329/159165208-3f35e896-3551-401b-bd78-adecb568a31f.png)


- The difficulty of computing stable and accurate neural networks: On the barriers of deep learning and Smale’s 18th problem ( PNAS 2022)
 - University of Cambridge, University of Oslo
 - cf. computability (Gödel and Turing)
 - **Question**: Why does DL lead to unstable methods and AI-generated hallucinations, even in scenarios where one can prove that stable and accurate neural networks exist?
   - There are well-conditioned problems where, paradoxically, mappings from training data to suitable NNs exist, but no training algorithm (even randomized) can compute approximations of the NNs from the training data.
   - Under specific conditions (e.g. MRI) there are algorithms that compute stable NNs for the problem in Eq. 1. These NNs, which we call fast iterative restarted networks (FIRENETs), converge exponentially in the number of hidden layers. Crucially, we prove that FIRENETs are robust to perturbations (Fig. 2, Bottom row), and they can even be used to stabilize unstable NNs (Fig. 3).
   
Fig 2.
![image](https://user-images.githubusercontent.com/38721329/159165410-69313feb-ec77-40a1-82a5-af464f558a9f.png)

Fig 3.
![image](https://user-images.githubusercontent.com/38721329/159165419-16476005-f8da-4b6e-9ca9-868c13037ff3.png)
